### PR TITLE
Add documentation on frontend controllers as application entrypoints

### PR DESCRIPTION
**Pull request by 94noni**
Some nice to have comments (for new comers), perhaps the wording is to be improved :s 
Ref https://symfony.com/doc/current/setup.html#the-symfony-demo-application

### CODE REVIEW

File: bin/console [hunk 1]
```diff
@@ -10,6 +10,15 @@ if (!is_file(dirname(__DIR__).'/vendor/autoload_runtime.php')) {
 
 require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
 
+/**
+ * This file is a Symfony Front Controller (entrypoint)
```
Comment (javiereguiluz) [inline bin/console:14]: Just asking: is it correct to use the term "front controller" when we are not in the HTTP/web context? Thanks.

Comment (94noni) [reply bin/console:14]: Well this is a good question I can not answer
Entrypoint is ok, perhaps we could just write this

the aim or this PR is to show the 2 way or booting symfony

Comment (94noni) [reply bin/console:14]: @javiereguiluz hello

in this page: https://symfony.com/doc/current/the-fast-track/en/3-zero.html#initializing-the-project

`The bin/ directory contains the main CLI entry point: console. You will use it all the time.`

so what do you think I rewrite: `This file is a Symfony CLI entrypoint`

File: bin/console [hunk 2]
```diff
@@ -10,6 +10,14 @@ if (!is_file(dirname(__DIR__).'/vendor/autoload_runtime.php')) {
 
 require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
 
+/**
+ * This file is a Symfony command-line entrypoint.
```
Comment (rosier) [inline bin/console:14]: How about also adding the word interface and shouldn't entrypoint be two words?

```suggestion
 * This file is a Symfony command-line interface (CLI) entry point.
```

Comment (94noni) [reply bin/console]: indeed, makes more sens to really document with full names thx

**Pull request closed without merging**